### PR TITLE
[DOCU-2416] Kong in Production> Topologies: Overview

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Summary

New Explanation doc: Overview of the three different deployment strategies
### Reason

[DOCU-2416](https://konghq.atlassian.net/browse/DOCU-2416)
### Testing

Questions:
- Am I missing any information? I feel like I'm missing some benefits/limitations, but couldn't find enough info anywhere.
- I didn't add anything to a "More information" section because I already added the relevant links to the main sections. Are there any links I should include in a section like that or should I just leave it out?

### CODE REVIEW

**Review by acgoldsmith — APPROVED** [review]
LGTM for actual copyedit; will defer to others WRT benefits/limitations :)

**Comment by windmgc** [comment]
https://docs.konghq.com/gateway/latest/reference/clustering/#main

I don't know whether this should be put into the "Topologies" category but it is surely a different way of deployment(different from the traditional/hybrid/dbless mode). It is based on the traditional mode and maybe it can be named as "traditional cluster".

From what I know there are very few users using this kind of deployment, and obviously, it should not be recommended by us

**Review by Guaris — APPROVED** [review]
LGTM merging after tests pass

**Pull request merged**
